Tests/HttpServerSpec/ActionsSpec/LogRequestsActionSpec.swift

```diff
@@ -9,14 +9,14 @@ class LogRequestsActionSpec: QuickSpec {
             var action: LogRequestsAction!
             var dataStorage: DataStorage!
             var request: HttpRequest!
-            var routesTable: RoutesTable!
+//            var routesTable: RoutesTable!
 
             beforeEach {
                 dataStorage = DataStorage()
                 dataStorage.addToRequestList(request: "PUT /form HTTP/1.1")
                 dataStorage.addToRequestList(request: "HEAD /requests HTTP/1.1")
-                routesTable = RoutesTable()
-                action = LogRequestsAction(routesTable: routesTable, dataStorage: dataStorage)
+//                routesTable = RoutesTable()
+                action = LogRequestsAction(dataStorage: dataStorage)
```

Comment:
Why are there commented out lines here?

Reply:
I meant to deleted all of them. I must have missed this set of action.